package.json

```diff
@@ -49,7 +50,7 @@
     "webpack-dev-server": "^1.8.2"
   },
   "peerDependencies": {
-    "react": ">=0.13.2 || ^0.14.0-rc1 || ^15.0.0-rc"
```

Comment:
Good point. Up to you, I can abandon this or just push it through if we want to exclude RC versions.